shared/js/ui/views/whitelist.es6.js

```diff
@@ -109,6 +106,12 @@ Whitelist.prototype = window.$.extend({},
       this.setup()
     },
 
+    _renderList: function () {
+      this.unbindEvents()
+      this.$container.html(whitelistItemsTemplate(this.model.list))
+      this.setup()
```

Comment:
Need to unbind and setup events again, even if it's just the list, otherwise after rerendering the "add to whitelist" button won't trigger an event anymore